src/Core.zig

```diff
@@ -156,6 +156,22 @@ pub fn cursorShape(core: *Core) CursorShape {
     return core.internal.cursorShape();
 }
 
+pub fn joystickPresent(core: *Core, joystick: Joystick) bool {
+    return core.internal.joystickPresent(joystick);
+}
+
+pub fn joystickName(core: *Core, joystick: Joystick) ?[:0]const u8 {
+    return core.internal.joystickName(joystick);
+}
+
+pub fn joystickButtons(core: *Core, joystick: Joystick) ?[]const bool {
+    return core.internal.joystickButtons(joystick);
+}
+
+pub fn joystickAxes(core: *Core, joystick: Joystick) ?[]const f32 {
```

Comment:
Can you add docs to these clarifying what the values are? Like what is axes[0] vs. axes[1]?

Reply:
let's also add docs to all 3 of these functions indicating when they return null.

Reply:
These are joystick-dependent as I dont use the Gamepad API of GFLW so they are no real names.
Should I still add some documentation like "axis[0] refers to the first axis of the joystick etc"?